lib/config/folderconfiguration.go

```diff
@@ -235,6 +239,14 @@ func (f *FolderConfiguration) prepare() {
 	if f.MarkerName == "" {
 		f.MarkerName = DefaultMarkerName
 	}
+
+	if f.ModTimeWindowS < 2 && f.Path != "" {
+		if usage, err := disk.Usage(f.Filesystem().URI()); err != nil {
```

Comment:
I wonder if this does the right thing, as on android GOOS=android, not GOOS=linux

Reply:
I think `GOOS == linux` is true on android, so it should at least try to do something, however as I wrote I suspect it wont work due restrictions on android, so the wrapper would still need to set this config itself.

Reply:
We should verify if that is the case, I suspect it will work, as it works via a simple stat call, which we already do.